fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/AutoPartitionManager.java

```diff
@@ -250,6 +252,14 @@ private void createPartitions(
                         "Auto partitioning skip to create partition {} for table [{}] as the partition is exist.",
                         partition,
                         tablePath);
+            } catch (TooManyPartitionsException t) {
+                LOG.warn(
+                        "Auto partitioning skip to create partition {} for table [{}], "
+                                + "because exceed the maximum number of partitions.",
+                        partition,
+                        tablePath);
+            } catch (FlussRuntimeException f) {
+                LOG.warn(f.getMessage());
             }
 
             currentPartitions.add(partition.getPartitionName());
```

Comment:
We should move this into the above try catch, otherwise, the state of `currentPartitions` is not correct.